client/components/remove-button/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+export class RemoveButton extends React.Component {
+	static propTypes = {
+		onRemove: PropTypes.func.isRequired,
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
```

Comment:
I don't think this is needed -- `localize` ought to handle supplying `this.props.translate` prior to all lifecycle methods.

Reply:
I was thinking it was useful for the unit tests, since they load the component directly, instead of the wrapped export.

Without the optional default `translate`, each test needs to explicitly pass an `identity` function, like:

`const wrapper = shallow( <RemoveButton onRemove={ noop } translate={ identity } /> );`

That seems a bit more cluttered to me, but I can also see how it makes the primary usage via `localize()` a bit more clearly defined. I'll go ahead and move that over to the tests.

Reply:
@jblz: I made both of those changes in https://github.com/Automattic/wp-calypso/pull/18064/commits/cfc24ddeddab526d1dfc22816b7e17c57230ba4f. I'll squash it before merging, but figured it'd be easier to review if you could see them in isolation, rather than getting buried in a rebase.

Reply:
> I'll squash it before merging, but figured it'd be easier to review if you could see them in isolation, rather than getting buried in a rebase.

Thanks -- this is definitely the preferred approach.